README.md

```diff
@@ -31,7 +31,7 @@ For notes on how to set up a production instance, see the [hosting documentation
 
 ## Development
 
-In order to run this software you need to set up a [Postgres 14](https://www.postgresql.org/) database.
+In order to run this software you need to set up a [Postgres 14](https://www.postgresql.org/) database. You will also need a working copy of `python3`
```

Comment:
I suppose this could be qualified to say "if you wish to make SQL changes and lint them locally."